src/OrchardCore/OrchardCore.Application.Cms.Targets/OrchardCore.Application.Cms.Targets.csproj

```diff
@@ -33,6 +33,7 @@
     -->
 
     <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.Admin\OrchardCore.Admin.csproj" PrivateAssets="none" />
+    <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.AdminTrees\OrchardCore.AdminTrees.csproj" PrivateAssets="none" />
```

Comment:
Do you need to add the feature in the recipes ?

Reply:
Not sure if this should be enabled by default. Just in case you think it's OK, I added a last commit.
It is adding an admin tree to the Blog recipe. Adds a menu for the blog and adds a list of content types menus to the content items menu:

![seed-tree-blog-recipe](https://user-images.githubusercontent.com/2589629/47514665-408a8200-d881-11e8-9c13-45acdb4b2700.jpg)

There is an issue about the blog editing page being difficult to find. This should fix it for the Blog recipe.